### PR TITLE
OHM-360: Added timezone support for polling crontab/about page

### PR DESCRIPTION
**Pull request by BMartinos**
This PR is dependant on https://github.com/jembi/openhim-core-js/pull/805
Adds timezone text to the console (Polling cronjob/about page)

### CODE REVIEW

File: app/scripts/controllers/channelsmodal.js [hunk 1]
```diff
@@ -339,7 +341,31 @@ app.controller('ChannelsModalCtrl', function ($scope, $modalInstance, $timeout,
 });
 
 // nested controller for the channel basic info tab
-app.controller('channelBasicInfoCtrl', function ($scope, $timeout, Api, Notify, Alerting) {
+app.controller('channelBasicInfoCtrl', function ($scope, $timeout, $interval, Api, Notify, Alerting) {
+
+
+  var updateTime = function(timezone) {
+    $scope.aboutInfo.serverTime = getTimeForTimezone(timezone);
+    $scope.aboutInfo.serverTimezoneOffset = getTimezoneOffset(timezone);
```
Comment (napiergit) [inline app/scripts/controllers/channelsmodal.js]: This probably doesn't need to be called every second?

File: app/scripts/controllers/channelsmodal.js [hunk 2]
```diff
@@ -339,7 +341,31 @@ app.controller('ChannelsModalCtrl', function ($scope, $modalInstance, $timeout,
...
+    }, 1000);
+  };
+
+  var error = function(err) {
+    Alerting.AlertAddServerMsg(err.status);
+  };
+
+  Api.About.get(success, error);
+
+
```
Comment (napiergit) [inline app/scripts/controllers/channelsmodal.js]: Do we not need to cancel the interval on destroy like you did in about controller?  Is it different for modal?

**Comment by napiergit** [comment]
Looks good @BMartinos.  Just two minor comments.

**Comment by BMartinos** [comment]
Thanks @napiergit well spotted with the comments above. I have corrected them if you can review again please.

**Comment by napiergit** [comment]
LGTM :+1:

**Pull request merged**